test/sharness/t0020-service-startup-init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+#
+# MIT LICENSED
+#
+test_description="Test service startup and init functionality"
+
+. lib/test-lib.sh
+
+test_expect_success "ipfs is installed on this machine" '
+   ipfs help >ipfs_help.txt &&
+   egrep -i "^Usage" ipfs_help.txt >/dev/null  
+'
+
+test_expect_success "launch ipfs daemon" '
+   mkdir -p ../.test_ipfs &&
+   IPFS_PATH="../.test_ipfs" eval '"'"'ipfs init'"'"' &&
+   IPFS_PATH="../.test_ipfs" eval '"'"'ipfs daemon & echo $! >../dPID.txt'"'"' &&
+   sleep 2 
+'
+
+test_expect_success "test config folder exists" '
+    mkdir -p ../.test_config
+'
+
+test_expect_success "init cluster-service" '
+    ipfs-cluster-service -f --config ../.test_config init 2>service_init.txt &&
+    grep "configuration written" service_init.txt >/dev/null &&
```

Comment:
don't grep output info messages. Test that the actual file has been created. Test that it contains a private key and an ID. (maybe use `jq`).

Also, make a separate test to check that the `-f` does it's job and replaces an existing configuration.